changelogs/fragments/265-url-encode-user.yml

```diff
@@ -1,2 +1,4 @@
 bugfixes:
 - Ensure user email/login is url encoded when searching for the user (#264)
+trivial:
```

Comment:
I guess `trivial` entries are not included in the auto generated Ansible Release Notes (https://github.com/ansible-community/ansible-build-data/blob/main/6/CHANGELOG-v6.rst)?

Reply:
Indeed, trivial is not shown as stated [here](https://github.com/ansible-community/antsibull-changelog/blob/main/docs/changelogs.rst#changelog-fragment-categories)

Reply:
Cool. Thanks for the reference.